src/compile/repeat.ts

```diff
@@ -0,0 +1,204 @@
+import {Channel} from '../channel';
+import {CellConfig, Config} from '../config';
+import {FieldDef} from '../fielddef';
+import {FILL_STROKE_CONFIG} from '../mark';
+import {Repeat} from '../repeat';
+import {RepeatSpec} from '../spec';
+import {Dict, duplicate, flatten, vals} from '../util';
+import {VgData, VgEncodeEntry, VgLayout, VgScale} from '../vega.schema';
+import {applyConfig, buildModel} from './common';
+import {assembleData} from './data/assemble';
+import {assembleLayoutData} from './layout';
+import {Model} from './model';
+
+
+export type RepeatValues = {
```

Comment:
Repeater value?

(Let's name things such that Repeat = spec, repeater = {`repeat: 'column/row'}`?)